kong/plugins/grpc-gateway/deco.lua

```diff
@@ -89,6 +89,11 @@ local function get_proto_info(fname)
 
   local dir, name = pl_path.splitpath(pl_path.abspath(fname))
   local p = protoc.new()
+  p:addpath("/usr/include")
+  p:addpath("/usr/local/opt/protobuf/include/")
+  p:addpath("/usr/local/kong/lib/")
+  p:addpath("kong")
+
```

Comment:
it might be good time to abstract `protoc.new()` to Kong level so we can automatically add
common path before return to user. but we can do in seperate PR.